README.md

```diff
@@ -45,6 +45,9 @@ plaid.getCategories(plaid_env, callback);
 
 plaid.getInstitution(institution_id, plaid_env, callback);
 plaid.getInstitutions(plaid_env, callback);
+
+plaid.searchInstitutions({id: institutionId}, env, callback);
+plaid.searchInstitutions({p: plaidProduct, q: searchString}, env, callback);
```

Comment:
`p` and `q` aren't very descriptive. Could we use `product` and `query` instead?